community/mc/build

```diff
@@ -6,5 +6,7 @@
     --sysconfdir=/etc \
     --disable-nls
 
+patch -p1 < glib.patch
+
```

Comment:
I'd prefer the patch as close to the start of the file as possible. :+1:

Otherwise, LGTM.